docs/setup.md

```diff
@@ -11,82 +11,95 @@
     ![Define your Ticketing/ITSM Integrations](./res/config-wizard-03.png)
 
-5. Click **Finish** to complete the configuration process.
+6. Click **Finish** to complete the configuration process.
     ![All set](./res/config-wizard-04.png)
 
-## Configure the Threat Detection Integrations Parameters
+## Configure the Threat Detection Integration Parameters
 
-Following are some prerequisites for using this solution pack:
-Open "Investigate Outbreak" playbook from "10 - SP - Outbreak Response Framework" playbook collection and add following configuration parameters in set variable step for the "Threat Detection Integrations" that you have selected in configuration wizard.
+1. The following are some prerequisites for using this solution pack:
 
-- Fortinet FortiSIEM Config: 
-![Fortinet FortiSIEM Config](./res/fsm_config.png)
+2. Open the **Investigate Outbreak** playbook from the *10 - SP - Outbreak Response Framework* playbook collection.
+Add the following configuration parameters in the *Set Variable* step depending on the *Threat Detection Integrations* that you have selected in the **Configuration** wizard.
 
-- Fortinet FortiAnalyzer Config:
-![Fortinet FortiAnalyzer Config](./res/faz_config.png)
+    - Fortinet FortiSIEM Configuration:
 
-- Fortinet FortiGate Config:
-![Fortinet FortiGate Config](./res/fgt_config.png)
+        ![Fortinet FortiSIEM Config](./res/fsm_config.png)
 
-- IBM QRadar Config:
-![IBM QRadar Config](./res/qradar_config.png)
+    - Fortinet FortiAnalyzer Configuration:
 
-- Splunk Config:
-![Splunk Config](./res/splunk_config.png)
+        ![Fortinet FortiAnalyzer Config](./res/faz_config.png)
+
+    - Fortinet FortiGate Configuration:
+
+        ![Fortinet FortiGate Config](./res/fgt_config.png)
+
+    - IBM QRadar Configuration:
+
+        ![IBM QRadar Config](./res/qradar_config.png)
+
+    - Splunk Configuration:
+
+        ![Splunk Config](./res/splunk_config.png)
 
 ## Working with Outbreak Response Framework - Best Practices
-Apart from best practices if you want to add new "Threat Detection Integrations" or "Ticketing/ITSM Integrations" after configuring the "Threat Detection Integrations" or "Ticketing/ITSM Integrations" using the Configuration Wizard, perform following pointers
 
-1. Configure the new integration using the required credentials 
-2. If you want to add "Threat Detection Integrations" then added "Threat Detection Integrations" name "Threat Hunt Tools" picklist
-3. If you want to add "Ticketing/ITSM Integrations" then added "Ticketing/ITSM Integrations" name "Ticketing Tools" picklist.
-4. Run "Outbreak Response Framework" configuration wizard and select the old and new added integration either in "Threat Detection Integrations" or "Ticketing/ITSM Integrations" 
-5. Following global variables must never be modified or deleted:
+To add new *Threat Detection Integrations* or *Ticketing/ITSM Integrations* apart from the ones listed in the **Configuration Wizard**, consider the following pointers:
```

Comment:
This can be treated as advance settings and not best practices as it not applicable for all.

Reply:
Done.